components/indexer.py

```diff
@@ -10,9 +10,9 @@ class Indexer:
     piston_switch: wpilib.DigitalInput
 
     intake_arm_piston: wpilib.Solenoid
-    intake_main_motor: ctre.WPI_TalonSRX
-    intake_left_motor: wpilib.Spark  # Looking from behind the robot
-    intake_right_motor: wpilib.Spark  # Looking from behind the robot
+    intake_main_motor: wpilib.interfaces.SpeedController
```

Comment:
You are actually reading the limit switch on this TalonSRX below, so this should still be typed as `ctre.WPI_TalonSRX`.